src/opendr/perception/multimodal_human_centric/audiovisual_emotion_learner/algorithm/data_utils.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import librosa
+import numpy as np
+import cv2
+import torch
+from facenet_pytorch import MTCNN

```

Comment:
Why not reusing OpenDR face detectors? This adds one additional dependency to the toolkit.

Reply:
Addressed, switched to OpenDR retinaface implementation